src/functions/binaryForkResolve.se

```diff
@@ -42,7 +42,7 @@ extern subcurrency: [allowance:[address,address]:uint256, approve:[address,uint2
 
 extern forkResolveContract: [resolveFork:[int256]:int256]
 
-inset('../macros/refund.sem')
+inset('../macros/assertZeroValue.sem')
```

Comment:
That's fine, I just did an all-file search-and-replace.  I'm guessing these reporting-related files have been deleted altogether in your branch, right?